CHANGELOG.md

```diff
@@ -11,6 +11,7 @@
 
 - Updated the lithostratigraphy codelist table.
 - Upgraded to OpenLayers 7.
+- Removed DescriptionFacies and DescriptionLithological from lithology.
```

Comment:
REQ: Hier sollten die Spaltennamen, so wie sie in der DB sind, genannt werden. Vorschlag:
```suggestion
- Removed obsolete columns `facies_description_lay` and `lithological_description_lay` from layer table.
```